ivi-layermanagement-examples/simple-ivi-share/src/simple-ivi-share.c

```diff
@@ -185,6 +185,7 @@ handle_share_surface_damage(void *data, struct ivi_share_surface *share_surface,
         if (window->share_buffer.eglimage != NULL) {
             pfEglDestroyImageKHR(display->egl.egldisplay,
                                  window->share_buffer.eglimage);
+            ivi_share_surface_release_shared_name(window->share_surface, window->share_buffer.name);
```

Comment:
Commit message of this patch is entirely missing. Please send a proper commit message which explains your implementation.